features/customer_registration.feature

```diff
@@ -1,8 +1,10 @@
 Feature: Customer registration
+  @wip
```

Comment:
Don't forget to remove these if you're finished, the feature's done and you're sending a PR